Add new integration Homee (#36526)
home-assistant/home-assistant.github.io: source/_integrations/homee.markdown

---
title: Homee
description: Instructions on how to setup Homee devices in Home Assistant.
ha_category:
  - Cover
ha_config_flow: true
ha_release: 2025.2
ha_iot_class: Local Push
ha_codeowners:
  - '@Taraman17'
ha_domain: homee
ha_platforms:
  - cover
ha_integration_type: hub
---

[Homee](https://hom.ee) is a smart home system, able to integrate various protocols such as Z-Wave, Zigbee, EnOcean, and more. The Homee {% term integration %} will let you use the {% term device devices %} from your Homee in Home Assistant.

There is currently support for the following device types in Home Assistant:

> - cover

Author: More platforms are being added before the first release will be in HA.

---
title: Homee
description: Instructions on how to setup Homee devices in Home Assistant.
ha_category:
  - Cover
ha_config_flow: true
ha_release: 2025.2
ha_iot_class: Local Push
ha_codeowners:
  - '@Taraman17'
ha_domain: homee
ha_platforms:
  - cover
ha_integration_type: hub
---

[Homee](https://hom.ee) is a smart home system, able to integrate various protocols such as Z-Wave, Zigbee, EnOcean, and more. The Homee {% term integration %} will let you use the {% term device devices %} from your Homee in Home Assistant.

There is currently support for the following device types in Home Assistant:

- cover

More {% term platform platforms %} will follow soon, as integration moves to core step by step.

## Prerequisites

You need to create a new user in your Homee that is only used by Home Assistant. For security reasons, it's recommended to:

- Use a strong, unique password.

> - Limit the user's permissions to only what's necessary for Home Assistant.

Reviewer: What is necessary?

Author: Hmm, maybe "what is needed"?
Or leave it out completely.
What is meant is, that you only give it the rights for what you intend to do from homeassistant. Maybe you want to limit access to some devices for whatever reason...

---
title: Homee
description: Instructions on how to setup Homee devices in Home Assistant.
ha_category:
  - Cover
ha_config_flow: true
ha_release: 2025.2
ha_iot_class: Local Push
ha_codeowners:
  - '@Taraman17'
ha_domain: homee
ha_platforms:

>   - cover

Reviewer: ```suggestion
  - cover
  - sensor
```

ha_integration_type: hub
---

> [Homee](https://hom.ee) is a smart home system, able to integrate various protocols such as Z-Wave, Zigbee, EnOcean, and more. The Homee {% term integration %} will let you use the {% term device devices %} from your Homee in Home Assistant.

Reviewer 2: ```suggestion
[Homee](https://hom.ee) is a smart home system, able to integrate various protocols such as Z-Wave, Zigbee, EnOcean, and more. The Homee {% term integration %} will let you use the {% term devices %} from your Homee in Home Assistant.
```

There is currently support for the following device types in Home Assistant:

- cover
- sensor

> More {% term platforms %} will be added soon.

Reviewer: ```suggestion
```

> More {% term platform platforms %} will follow soon, as integration moves to core step by step.

Reviewer 2: ```suggestion
More {% term platforms %} will be added soon.
```
I would remove this implementation detail.

Author: I added it, because coderabbit hinted to do so, because I mention various protocols in the first paragraph.

Author: Now I saw, which detail you meant...

## Prerequisites

> You need to create a new user in your Homee that is only used by Home Assistant. For security reasons, it's recommended to:

Reviewer 2: ```suggestion
You need to create a new user in your Homee app. Create a user that is only used by Home Assistant. For security reasons, it's recommended to:
```

- Use a strong, unique password.

> - Set the user's permissions for what you what to do from Home Assistant.

Reviewer 2: ```suggestion
- Restrict the user's permissions for what you want to do from Home Assistant.
```
just an idea... "restrict" sounds more specific to me. But feel free to ignore the comment.

- Do not use this account for any other purpose.

> 1. In your Homee app tap the menu button in the top-left corner.

Reviewer 2: ```suggestion
1. In your Homee app, in the top-left corner, tap the menu button.
```

> 2. Tap the gear wheel icon on the top right of the opening menu.

Reviewer 2: ```suggestion
2. On the top right of the opening menu, tap the gear {% icon "mdi:gear-outline" %} icon.
```
Location before action. 
Gear: we have an icon for this :tada: , in case it matches what you have in your software.
![image](https://github.com/user-attachments/assets/5dc2d9fd-85ea-4a27-9937-d75b1cfef3db)

> 3. Choose "Manage users".

Reviewer 2: ```suggestion
3. Choose **Manage users**.
```
We use bold to markup UI strings

> 4. Choose "Add user" and configure the desired values.

Reviewer 2: ```suggestion
4. Choose **Add user** and configure the desired values.
```

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
  description: The IP Address of your Homee.
User:

>   description: The username you chose in the prerequisites.

Reviewer 2: ```suggestion
  description: The username you chose in the Prerequisites.
```
